tests/framework/CodeInterfaceTests/Neutrino/gold/Neutrino_Base/sampleNeutrino/2/results.csv

```diff
@@ -0,0 +1,301 @@
+time,result
+0.0241607,0.0266296

```

Comment:
you should remove this... you are not testing against this... you just test _"Neutrino_Base/out_dump.csv"_ in your **tests** file